src/main.c

```diff
@@ -514,6 +516,11 @@ usage()
 	printf("\tInstall the second VIA chip expansion at $9F10\n");
 	printf("-testbench\n");
 	printf("\tHeadless mode for unit testing with an external test runner\n");
+	printf("-mhz <integer>\n");
+	printf("\tRun the emulator with a system clock speed other than the default of\n");
+	printf("\t8 MHz. Valid values are in the range of 1-40, inclusive. This option\n");
```

Comment:
this seems fine to me as it is right now, I mean the default is 8 so 1 mhz is already at 1/8th speed. But in general, do we perhaps want a float for this?  So that we can go even lower , into the Kiloherz range?

Reply:
There's some integer math on MHZ in audio.c, but other than that it otherwise could possibly behave like a float.

Reply:
I wonder who ever is going to use that though, <1 mhz... could always change it if someone actually requests it